app/components/scroll-btn.js

```diff
@@ -37,7 +37,7 @@ class ScrollBtn extends Component {
     const { show } = this.state;
     return (
       <div className={`back-top ${show ? 'show' : ''}`}>
-        <a className="go-to-anchor" href="#hero">
+        <a className="go-to-anchor" href="#menu">
```

Comment:
Updated to go to menu instead. Imho this type of button should take the user to the main menu at the top of the page. Also in past editions, there is no hero image, so the scroll up button was not working.